waku/standards/core/14/message.md

```diff
@@ -76,14 +82,14 @@ This attribute MAY contain the Unix epoch time in nanoseconds.
 If the attribute is omitted, it SHOULD be interpreted as timestamp 0.
 
 - The `ephemeral` attribute, if present, signifies the transient nature of the message.
-For example, an ephemeral message SHOULD not be persisted by the Waku network.
+For example, an ephemeral message SHOULD not be persisted by the [64/WAKU2-NETWORK](/waku/standards/core/64/network.md).
```

Comment:
`64` describes a specific instantiation of a Waku network and this RFC is more general.

```suggestion
For example, an ephemeral message SHOULD not be persisted by other nodes on the same network.
```